lib/amazing_print/custom_defaults.rb

```diff
@@ -7,8 +7,8 @@ class << self
     # Class accessor to force colorized output (ex. forked subprocess where TERM
     # might be dumb).
     #---------------------------------------------------------------------------
-    def force_colors!(value = true)
-      @force_colors = value
+    def force_colors!(colors: true)
```

Comment:
Won't this API change break some code?

Reply:
Yeah I suppose you're right. Should we put this back?

Reply:
We also can do `AmazingPrint.force_colors = false` since it's an attribute accessor so we really don't need `.force_colors!` at all since it's really redundant.